webpack.config.js

```diff
@@ -117,9 +117,14 @@ function test () {  // eslint-disable-line
   return Object.assign(production(), watchOptions())
 }
 
-function merge (config, env) {
-  var merged = Object.assign({}, env, config)
-  merged.plugins = (config.plugins || []).concat(env.plugins || [])
+function merge (config, envConfig) {
+  var merged = Object.assign({}, envConfig, config)
+  merged.plugins = (config.plugins || []).concat(envConfig.plugins || [])
+  // In development we overwrite the output path so the dev server serves everything from the gen subdir
+  if (env === 'development') {
+    // Note that this directory will never be created since the dev server does not write out to files
+    merged.output.path = path.resolve(__dirname, 'app', 'extensions', 'gen')
```

Comment:
weird, I will add to `.gitignore`.

that is expected and the same behavior as before (content base was previously set via `--content-base=./app/extensions/brave"` in the webpack-dev-server params)